vllm/model_executor/layers/pooler.py

```diff
@@ -28,15 +31,41 @@ class Pooler(nn.Module):
         normalize: Whether to normalize the pooled data.
     """
 
-    def __init__(self,
-                 pooling_type: PoolingType,
-                 normalize: bool,
-                 softmax: bool = False):
+    def __init__(
+        self,
+        pooling_type: Optional[PoolingType] = None,
+        normalize: Optional[bool] = None,
+        softmax: Optional[bool] = None,
```

Comment:
```suggestion
        pooling_type: PoolingType,
        normalize: bool,
        softmax: bool,
```

The original arguments (pooling type, normalize, softmax) should not be `None` as they have already been overridden.